seqr/views/apis/data_manager_api_tests.py

```diff
@@ -851,11 +891,16 @@ def mock_write(content):
                 # test loading new data without deleting existing data
                 data = [params['no_existing_data']]
                 body.pop('mappingFile')
-                _test_basic_data_loading(data, 1, 1, '1kg project nåme with uniçøde', 1, 2, 0)
+                _test_basic_data_loading(data, 1, 1, 2, body)
 
                 # Test loading data when where are duplicated individual ids in different projects.
-                data = [params['reused_indiv_id_data']]
-                _test_basic_data_loading(data, 1, 1, 'Test Reprocessed Project', 1, 20, 0)
+                data = params['duplicated_indiv_id_data']
+                mock_writes = []
+                _test_basic_data_loading(data, 2, 2, 20, body,
+                                         project_names='1kg project nåme with uniçøde, Test Reprocessed Project',
+                                         num_created_samples=2)
+                self.assertSetEqual({re.sub(r'^S[0-9]*', 'S', s) for s in mock_writes}, params['write_data'])
```

Comment:
would be neater to mock sample guid creation instead, but this is okay

Reply:
It must be doable.